l10n/data/uk.json

```
@@ -0,0 +1,550 @@
+        "Використовуйте код чи назву валюти: ",
+        "`@INCS2bot price {валюта}`",
+        "(наприклад, `@INCS2bot price uah гривня` чи `@INCS2bot price гривня`)"
+    ],
+    "exchangerate_inline_title_selected": "Дізнатись ціну ключа в {}",
+    "exchangerate_inline_description_selected": "1x Ключ від кейсу в Counter-Strike коштує {} {}",
+    "exchangerate_inline_text_selected": "1x Ключ від кейса в Counter-Strike коштує {} {}",
+    "exchangerate_inline_title_notfound": "Нічого не знайдено!",
+    "exchangerate_inline_description_notfound": "Натисніть сюди, щоб дізнатись усі доступні валюти",
+    "game_status_button_title": "Стан серверів",
+    "game_status_inline_title": "Стан серверів",
+    "game_status_inline_description": "Перевірити доступність серверів",
+    "game_status_text": [
+        "{} **Стан служб Counter-Strike:**",
+        "",
+        "• Ігровий координатор: {}",
+        "• Підключення до серверів: {}",
+        "• Планувальник матчмейкінгу: {}",
+        "• Інвентарі гравців: {}",
+        "• Steam веб-API: {}"
+    ],
+    "game_version_button_title": "Версія гри",
+    "game_version_inline_title": "Версія гри",
+    "game_version_inline_description": "Перевірити останню версію гри",
+    "game_version_text": [
+        "⚙️ Актуальна версія гри:",
+        "",
+        "CS:GO: `{}` `({})`",
+        "Останнє оновлення: {}",
+        "",
+        "CS2: `{}` `({})`",
+        "Останнє оновлення: {}",
+        "",
+        "ℹ️ Трекер оновлень Counter-Strike: [@cstracker](http://t.me/cstracker/7)"
+    ],
+    "game_dropcap_button_title": "Оновлення обмежень",
+    "game_dropcaptimer_inline_title": "Оновлення обмежень",
+    "game_dropcaptimer_inline_description": "Час до оновлення обмежень щотижневого досвіду та дропу",
+    "game_dropcaptimer_text": "⏳ Час до оновлення обмежень щотижневого досвіду та дропу: {} д. {} год. {} хв. {} сек.",
+    "gun_button_text": "Дані о зброї",
+    "gun_select_category": "#️⃣ Виберіть категорію, котра Вас цікавить:",
+    "gun_pistols": "Пістолети",
+    "gun_select_pistol": "🔫 Виберіть пістолет:",
+    "gun_heavy": "Важке Озброєння",
+    "gun_select_heavy": "🔫 Виберіть важку зброю:",
+    "gun_smgs": "Пістолети-кулемети",
+    "gun_select_smg": "🔫 Виберіть пістолет-кулемет:",
+    "gun_rifles": "Гвинтівки",
+    "gun_select_rifle": "🔫 Виберіть гвинтівку:",
+    "gun_summary_text": [
+        "🗂 Детальна інформація про {}:",
+        "",
+        "• Походження: {}",
+        "• Вартість: {}$",
+        "• Обойма: {}/{}",
+        "• Темп вогню: {} в/хв.",
+        "• Нагорода за вбивство: {}$",
+        "• Мобільність: {} од.",
+        "",
+        "• Кулепроникність: {}%",
+        "• Дальність поражения (стоя / сидя): {} / {} м",
+        "",
+        "• Час готовності зброї: {} сек.",
+        "• Перезарядка: {} / {} сек.",
+        "(готовність обойми / готовність вогню)",
+        "",
+        "💢 Інформація про урон:",
+        "(суперник в броне / без броні)",
```

```suggestion
        "(противник/супротивник у броні / без броні)",
```
Я лично не могу определится как лучше будет, по этому открыт к дискусии

---

ворог у броні
золотая середина